selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java

```diff
@@ -566,9 +566,7 @@ public void runActionForTabFromContextMenu(TabActionLocator tabAction) {
    * @param text text which should be typed
    */
   public void typeTextIntoEditor(String text) {
-    loader.waitOnClosed();
```

Comment:
`loader.waitOnClosed();` is needed here to be sure the loader is not displayed in front of editor, or it is gone after the text has been pasted.

Reply:
loader.waitOnClosed() does at least a one second wait. On this one, I am not sure, but I'd like to try removing it. We can reenable it if the test start failing.
Generally, I think this is the wrong approach. We should be testing for conditions that we need, for example we want the editor to have keyboard focus.